lua/entities/gmod_wire_holoemitter.lua

```diff
@@ -66,7 +70,7 @@ if CLIENT then
 			return true
 		end
 
-		if not next(self.Points) then return true end
+		if table.IsEmpty(self.Points) then return true end
```

Comment:
fun fact, the source code for `table.IsEmpty` used to be absolutely horrible back when this code was written. using `not next` was infinitely faster. these days, the source code for `table.IsEmpty` is literally just `not next` https://github.com/Facepunch/garrysmod/blob/master/garrysmod/lua/includes/extensions/table.lua#L55-L61